src/commands/auth/api/user.ts

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+
+const BACKEND_URL = "https://api-staging.anyflow.pro/api";
+
+// Make a request to get user information
+export async function getUserResponse(token:string) {
+    const response = await axios.get(`${BACKEND_URL}/user`, {
```

Comment:
if axios fails it will throw. 
but status handling expects it to not throw.

please test all possible statuses using https://httpstat.us/